connect/src/com/telenor/connect/utils/ConnectUrlHelper.java

```diff
@@ -60,6 +79,34 @@ public static Uri getAuthorizeUri(
         return builder.build();
     }
 
+    public static Uri getMobileConnectAuthorizeUri(
```

Comment:
This is almost identical to the getConnectAuthorizeUri. Can't you move the .appendPath(OAUTH_PATH).appendPath("authorize") into the ConnectSdkProfile and make them identical?